src/main/resources/swagger/api-docs.yaml

```diff
@@ -1034,7 +1034,7 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorReport'
-  /api/google/v1/petServiceAccount/{project}/{email}:
+  /api/google/v1/petServiceAccount/{project}/{email}/key:
```

Comment:
I noticed this API was inconsistent: it returns a key, but all the other APIs that return keys have the `/key` suffix. So I updated the swagger definition, but made the server code handle both patterns (with `/key` and without). I think this should be safe, but let me know if you disagree.

Reply:
this will change the generated client, I would leave it alone

Reply:
ok, removed